mobile-app/app/screens/AppNavigator/screens/Balances/BalancesScreen.tsx

```diff
@@ -79,36 +88,50 @@ export function BalancesScreen ({ navigation }: Props): JSX.Element {
       <Announcements />
       <BalanceControlCard />
       <ThemedView
-        style={tailwind('flex flex-row justify-between')}
+        light={tailwind('bg-white')}
+        dark={tailwind('bg-gray-800')}
+        style={tailwind('mx-2 my-4 p-4 rounded-lg flex flex-row justify-between items-center')}
+        testID='total_portfolio_card'
       >
-        <ThemedSectionTitle
-          testID='balances_title'
-          text={translate('screens/BalancesScreen', 'PORTFOLIO')}
-        />
+        <View>
+          <ThemedText
+            light={tailwind('text-gray-500')}
+            dark={tailwind('text-gray-400')}
+            style={tailwind('text-base text-gray-500')}
+          >
+            {translate('screens/BalancesScreen', 'Total Portfolio Value')}
+          </ThemedText>
+          <NumberFormat
+            decimalScale={8}
+            displayType='text'
+            prefix='$'
+            renderText={(value) =>
+              <BalanceText
+                dark={tailwind('text-gray-200')}
+                light={tailwind('text-black')}
+                style={tailwind('mr-2 flex-wrap text-2xl font-bold')}
+                testID='total_usd_amount'
+                value={value}
+              />}
+            thousandSeparator
+            value={totalUsdValue.toFixed(2)}
+          />
+        </View>
         <ThemedTouchableOpacity
           testID='toggle_balance'
           light={tailwind('bg-transparent')}
           dark={tailwind('bg-transparent')}
-          style={tailwind('flex flex-row pt-4 pr-4 items-center')}
           onPress={onToggleDisplayBalances}
         >
           <ThemedIcon
             iconType='MaterialIcons'
-            dark={tailwind('text-darkprimary-500')}
-            light={tailwind('text-primary-500')}
-            style={tailwind('self-center pr-1')}
+            dark={tailwind('text-darkprimary-500 border-gray-700')}
+            light={tailwind('text-primary-500 border-gray-200')}
+            style={tailwind('border rounded p-1.5')}
```

Comment:
Need to handle the overflow styling, this is from iOS
![IMG_A0498D32464B-1](https://user-images.githubusercontent.com/40191153/149120716-8e57d7a4-1016-4cd8-8ce7-d61105a5e33f.jpeg)

Reply:
From the same image, notice the toggle balance icon border is not visible on iOS